Makefile

```diff
@@ -1,25 +1,37 @@
-TARGET        = phpbrew
-SIGNATURE     = $(TARGET).asc
+# See https://tech.davis-hansson.com/p/make/
+MAKEFLAGS += --warn-undefined-variables
+MAKEFLAGS += --no-builtin-rules
+
+PHPBREW_PHAR        = phpbrew
+SIGNATURE     = $(PHPBREW_PHAR).asc
 CP            = cp
 INSTALL_PATH  = /usr/local/bin
 TEST          = phpunit
 
-$(TARGET): vendor $(shell find bin/ shell/ src/ -type f) box.json.dist .git/HEAD
-	box compile
-	touch -c $@
+PHAR_SRC_FILES := $(shell find bin/ shell/ src/ -type f)
+
+
+.DEFAULT_GOAL := help
+
+
+.PHONY: help
+help:
+	@printf "\033[33mUsage:\033[0m\n  make TARGET\n\n\033[32m#\n# Commands\n#---------------------------------------------------------------------------\033[0m\n\n"
+	@fgrep -h "##" $(MAKEFILE_LIST) | fgrep -v fgrep | sed -e 's/\\$$//' | sed -e 's/##//' | awk 'BEGIN {FS = ":"}; {printf "\033[33m%s:\033[0m%s\n", $$1, $$2}'
 
-vendor: composer.lock
-	composer install
-	touch $@
 
-.PHONY: sign
```

Comment:
the `sign` and `SIGNATURE` because now the PHAR is signed directly in the CI so there is no need for it in the Makefile anymore.